### ISSUE TITLE
[Bug] Severe Win32 problems [sf#275]

### ISSUE THREAD

**Issue by openMSX-import** [openMSX/openMSX#576]
**Reported by manuelbi on 2007-08-24 07:31 UTC**
There are several Win32 problems, which might be all related:
- 0.6.2 doesn't work on Windows 98 and ME, while 0.6.1 does. It either crashes, or hangs while doing nothing.
- since 0.6.2, openMSX doesn't give any stdout output anymore on Windows
- since 0.6.2, the above mentioned problems seem also to occur on Windows XP sometimes:

Jul 22 2007 21:57:46 *  GuyveR800 back with similar issue as last time
&lt;GuyveR800&gt; openMSX 0.6.2-dev6732 suddenly fails to
start
&lt;GuyveR800&gt; no crashes this time
&lt;GuyveR800&gt; I can start it without arguments
&lt;GuyveR800&gt; and I can start it with -machine Panasonic_FS-A1GT
&lt;GuyveR800&gt; but I can't start it with -machine
Boosted_MSX2_EN
&lt;AuroraMSX&gt; error msg?
&lt;GuyveR800&gt; none
&lt;GuyveR800&gt; starting with nothing but a -diska 
doesn't work either
&lt;GuyveR800&gt; 'openmsx': ok   'openmsx -diska test.dsk': not ok  'openmsx -diska test.dsk -machine Panasonic_FS-A1GT': fine
&lt;GuyveR800&gt; I used the 0.6.2 release earlier
&lt;GuyveR800&gt; that one crash 75% of the time
&lt;GuyveR800&gt; regardless of arguments
&lt;GuyveR800&gt; it gave windows GPF, or wouldn't start (same as now)
&lt;GuyveR800&gt; XP MCE SP2 NL
&lt;GuyveR800&gt; AthlonX2 3800+ 1GB
&lt;GuyveR800&gt; 0.6.1 no problems.. but I need the Gfx9000 fixes in 0.6.2, otherwise I'd downgrade

Again, I have the (maybe unfounded) feeling that this is all related, as it seems to got introduced at about the same time.

If someone could do a bisect to find out when this problem started to occur (which SVN commit), it would be a whole lot easier to fix it.

**Comment by openMSX-import** [comment]
**Commented by manuelbi on 2007-09-07 21:33 UTC**
Logged In: YES 
user_id=78178
Originator: YES

Problem 2 and 3 seem to be fixed when using the latest libraries and with the fixes in SVN that switch -mwindows to -mconsole.

The first issue is somewhat strange. On some people's machines it doesn't work at all (see http://www.msx.org/forumtopicl7746.html ) and on Wolf's Windows 98 machine, it seems to work OK, so far. I did have heavy problems with 0.6.2 on Windows ME.

**Comment by openMSX-import** [comment]
**Commented by manuelbi on 2007-12-25 13:55 UTC**
Logged In: YES 
user_id=78178
Originator: YES

The problems I had on Windows ME are because of a faulty init.tcl causing crashes (see other bug report about this: 1858050).

GuyveR800 reported that the above mentioned problems are gone in openMSX 0.6.3, in which we fixed a timer problem in Win32 specific code that goes bananas when there are hyper threading or multi core CPU's in the host PC.

Closing for now.
